src/robothub_oak/application.py

```diff
@@ -10,6 +10,7 @@
 
 import robothub_core
 from depthai_sdk import OakCamera
+from watch_dog import *
```

Comment:
This import statement will fail. Please use either relative Python 3 import: `from .watch_dog import *` or more consistent with already existing imports `from robothub_oak.watch_dogs import *`